CMake Error

On building with `cmake` with version **3.16.3**, build process crashes with this output:

```
CMake Error at CMakeLists.txt:56 (set_target_properties):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "CXX_STANDARD" is not allowed.


-- Configuring incomplete, errors occurred!
```

[Line 56, where the fault points to](https://github.com/giacomo-b/CppRobotics/blob/b179c953d0043388a2a82b4d66493f4412975e91/CMakeLists.txt#L56)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,8 +53,6 @@ file(GLOB_RECURSE headers CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/include
 
 add_library(Robotics INTERFACE)
 
-set_target_properties(Robotics PROPERTIES CXX_STANDARD 17)
-
 # being a cross-platform target, we enforce standards conformance on MSVC
 target_compile_options(Robotics INTERFACE "$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/permissive->")
 

